pytorch_lightning/strategies/single_tpu.py

```diff
@@ -83,5 +83,4 @@ def register_strategies(cls, strategy_registry: Dict) -> None:
 
     def teardown(self) -> None:
         super().teardown()
-        # TPU teardown
-        os.environ.pop("PT_XLA_DEBUG", None)
+        self.accelerator.teardown()
```

Comment:
I think we can remove this override? The TPU tear down logic is in accelerator, single device and tpu single device shares the same teardown logic now